pkg/gpt/gpt.go

```diff
@@ -65,6 +66,29 @@ type GPT struct {
 	Parts []Part
 }
 
+// PartitionTable defines all the partition table information.
+// This includes the MBR and two GPTs. The GPTs are
+// similar but not identical, as they contain "pointers"
+// to each other in the BackupLBA in the Header.
+// The design is defective in that if a given Header has
+// an error, you are supposed to just assume that the BackupLBA
+// is intact, which is a pretty bogus assumption. This is why
+// you do standards like this in the open, not in hiding.
+// I hope someone from Intel is reading this.
+type PartitionTable struct {
+	MasterBootRecord *MBR
+	Primary          *GPT
+	Backup           *GPT
+}
+
+func (m *MBR) String() string {
+	b, err := json.MarshalIndent(m, "", "\t")
```

Comment:
Won't you just get a full array dump of the bytes in the MBR this way?

Reply:
yeah, and that's fine. Most of the mbr is an x86 binary. And you don't want to lose those bytes.